design-tokens/tokens/layer3/dark.json

```diff
@@ -1 +1,19 @@
-{}
\ No newline at end of file
+{
+  "echoes": {
+    "overlays": {
+      "innerShadow": {
+        "default": {
+          "value": {
+            "x": "0",
+            "y": "-6",
+            "blur": "10",
+            "spread": "0",
+            "color": "rgba(255,255,255,0.1)",
+            "type": "innerShadow"
+          },
+          "type": "boxShadow"
+        }
+      }
```

Comment:
@marciopmoreira6 
We have a technical issue:
We can't use a regular box-shadow. Instead we're using a gradient as background for an extra element.
Is it possible for you to create a token for this?

Example (this was for us to test):
top : `radial-gradient(farthest-side at 50% 0, rgba(0, 0, 0, 0.05), rgba(0, 0, 0, 0)) center top`
bottom: `radial-gradient(farthest-side at 50% 100%, rgba(0, 0, 0, 0.05), rgba(0, 0, 0, 0)) center bottom`

Reply:
Discussed live with @gregaubert ✅